src/moonbeam/common.js

```diff
@@ -135,14 +111,16 @@ export const scheduleTask = async ({
         ? { Fixed: { executionTimes: [nextExecutionTime, twoTimeSlotsTimestamp] } }
         : { Fixed: { executionTimes: [0] } };
 
+    console.log('oakAsset.location: ', JSON.stringify(oakAsset.location));
     const sendExtrinsicPromise = Sdk().scheduleXcmpTimeTaskWithPayThroughRemoteDerivativeAccountFlow({
         oakAdapter,
         destinationChainAdapter: moonbeamAdapter,
         taskPayloadExtrinsic,
-        scheduleFeeLocation: moonbeamDefaultAsset.location,
-        executionFeeLocation: moonbeamDefaultAsset.location,
-        scheduleAs: u8aToHex(keyringPair.addressRaw),
+        scheduleFeeLocation: oakAsset.location,
+        executionFeeLocation: moonbeamAsset.location,
         keyringPair: moonbeamKeyringPair,
+        invoker: moonbeamAdapter,
+        invokeFeeLocation: oakAsset.location,
```

Comment:
Add the 'invokeFeeLocation' parameter to specify the asset for the XCM message fee when invoking the scheduled task.

Add the 'invokeFeeLocation' parameter to specify the invoker chain.

Reply:
let’s use `caller` instead of `invoker`, and `callerXcmFeeLocation` to replace `invokeFeeLocation`

Reply:
I don't know my pr code
I need pr code Oak